### PR TITLE
INIT fixes.

### PR DESCRIPTION
**Pull request by whyjustin**

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -1,63 +1,86 @@
...
 # Grant Default CI Account Edit Access to All Projects and OpenShift Project
-oc policy add-role-to-user edit system:serviceaccount:${OSE_CI_PROJECT}:default -n ${OSE_CI_PROJECT}
+oc policy add-role-to-user edit system:serviceaccount:${OSE_NEXUS_PROJECT}:default -n ${OSE_NEXUS_PROJECT}
 
-# Process Nexus Template
-echo
-echo "Processing Nexus OSS Template..."
-echo
-oc create -f "${SCRIPT_BASE_DIR}/nexus-oss.json" -n ${OSE_CI_PROJECT} >/dev/null 2>&1
+if [[ ${VERSION} == "oss" ]]
```
Comment (whyjustin) [inline init.sh]: Seems you can only have one installation per project as they both attempt to create an rhel7 image stream.

**Comment by whyjustin** [comment]
Getting a lot of errors in the logs when following these scripts.

File: init.sh (hunk 1 above)
Comment (punkisdead) [inline init.sh]: > Seems you can only have one installation per project as they both attempt to create an rhel7 image stream.

Maybe we need to separate the creation of the rhel7 image stream from the creation of the nexus oss/pro templates?

Comment (vap0rtranz) [inline init.sh]: Shouldn't need to if it's RHEL based Docker.

I'm going to try switching to new-app + expose svc via local registry to see if that works.

**Comment by whyjustin** [comment]
Merged to ose via 50f1a51.

**Pull request closed without merging**